index.html

```diff
@@ -15,68 +16,87 @@ <h2 class="title">INTENTION TIMER</h2>
         <h2>New Activity</h2>
         <section id="activityForm" class="form">
           <label class="category"><h3>Select a category:</h3></label>
-          <div class="button-box">
-            <button id="study" class="study"><img class="study-active hidden" src="./assets/study-active.svg"><img class="study-image"src="./assets/study.svg">Study</button>
-            <button id="meditate" class="meditate"><img class="meditate-active hidden" src="./assets/meditate-active.svg"><img class="meditate-image" src="./assets/meditate.svg">Meditate</button>
-            <button id="exercise" class="exercise"><img class="exercise-active hidden" src="./assets/exercise-active.svg"><img class="exercise-image" src="./assets/exercise.svg">Exercise</button>
+          <div id="buttonBox" class="button-box">
+            <button id="study" class="study">
+              <img id="studyActive" class="study-active hidden" src="./assets/study-active.svg" alt="Study Selected">
+              <img id="studyImage" class="study-image"src="./assets/study.svg" alt="">Study
+            </button>
+            <button id="meditate" class="meditate">
+              <img id="meditateActive" class="meditate-active hidden" src="./assets/meditate-active.svg" alt="Meditate Selected">
+              <img id="meditateImage" class="meditate-image" src="./assets/meditate.svg" alt="">Meditate
+            </button>
+            <button id="exercise" class="exercise">
+              <img id="exerciseActive" class="exercise-active hidden" src="./assets/exercise-active.svg" alt="Exercise Selected">
+              <img id="exerciseImage" class="exercise-image" src="./assets/exercise.svg" alt="">Exercise
+            </button>
           </div>
           <section class="error invisibility" id="buttonError">
-            <img class="error-image" src="./assets/warning.svg">
+            <img class="error-image" src="./assets/warning.svg" alt="Warning Error Select Activity Icon">
```

Comment:
Nice work getting the alt attributes in the images! Accessibility is important, as you will learn about in Mod 2